js/repositories.bzl

```diff
@@ -7,14 +7,10 @@ See https://docs.bazel.build/versions/main/skylark/deploying.html#dependencies
 load("//js/private:maybe.bzl", http_archive = "maybe_http_archive")
 
 def rules_js_dependencies():
-    "Dependencies for users of aspect_rules_js"
```

Comment:
Any specific reason for dropping this? It's not used/visible somewhere?

Reply:
Just cleanup as the equivalent macros in all other rule sets don't have the docstring comment. Its not used/visible anywhere on the docsite.

Reply:
👍